.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+/packages/app-store/applecalendar/**/* @calcom/Foundation
```

Comment:
Sadly CODEOWNERS does not support `/packages/app-store/*calendar` or some other dynamic syntax like that to always catch all calendar apps.